v3/src/components/inspector-panel.tsx

```diff
@@ -86,6 +86,16 @@ export const InspectorPalette = ({children, Icon, title, paletteTop = 0,  panelR
   const paletteRef = useRef<HTMLDivElement>(null)
   const inBounds = panelRect && isWithinBounds(paletteWidth, panelRect)
 
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    const { key } = e
+    switch (key) {
+      case "Escape":
+      case "Enter":
+        setShowPalette(undefined)
+        break
+    }
```

Comment:
For a single case like this, I'd be inclined to use an `if`:
```suggestion
    if (key === "Enter" || key === "Escape") {
      setShowPalette()
    }
```